### PR TITLE
chore: Use types

### PR DESCRIPTION
**Pull request by Hoopra**
Required as interface produces an error if compared to `Record<string, ...>` (Index signature is missing in type).

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -14,7 +14,8 @@ const cli = meow(
     --tinyIntAsBoolean, -b    Treat TinyInt as Boolean
     --binaryAsBuffer, -B      Treat Binary as Buffer
     --nullAsUndefined, -u     Treat null as undefined   
-    --nullPlusUndefined, -p   Treat as null and undefined   
+    --nullPlusUndefined, -p   Treat as null and undefined
+    --types                   Output types instead of interfaces
```
Comment (rasmussyberg) [inline src/index.ts]: I think it would be better with 
`--outputAsTypes, -t `

Comment (Hoopra) [reply src/index.ts]: `-t` is already taken above, but renamed to `outputAsTypes`

Comment (rasmussyberg) [reply src/index.ts]: Or just skip this part and just use types instead of interfaces. Not sure why anyone would care if it's one or the other

Comment (Hoopra) [reply src/index.ts]: it actually can make a difference. As I understand a `type` allows keys that are not specified (indexable) while `interface` does not, so `interface` is more strict

**Pull request merged**